core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -23,132 +23,139 @@ import { Injectable } from '@angular/core';
 import cloneDeep from 'lodash/cloneDeep';
 
 import { AppConstants } from 'app.constants';
+import { ICustomizationArgs } from
+  'domain/state/CustomizationArgsObjectFactory';
+import { IParamChangeBackendDict, ParamChange } from
+  'domain/exploration/ParamChangeObjectFactory';
+import { IParamSpecsBackendDict, ParamSpecs, ParamSpecsObjectFactory } from
+  'domain/exploration/ParamSpecsObjectFactory';
+import { Interaction } from 'domain/exploration/InteractionObjectFactory';
 import { LoggerService } from 'services/contextual/logger.service';
 import { ParamChangesObjectFactory } from
   'domain/exploration/ParamChangesObjectFactory';
-import { ParamSpecsObjectFactory } from
-  'domain/exploration/ParamSpecsObjectFactory';
-import { StatesObjectFactory } from 'domain/exploration/StatesObjectFactory';
+import { State } from 'domain/state/StateObjectFactory';
+import { StatesBackendDictMapping, States, StatesObjectFactory } from
+  'domain/exploration/StatesObjectFactory';
 import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service';
+import { Voiceover } from 'domain/exploration/VoiceoverObjectFactory';
 
 const INTERACTION_SPECS = require('interactions/interaction_specs.json');
 
+export interface IExplorationBackendDict {
+  /* eslint-disable camelcase */
+  author_notes: string;
+  blurb: string;
+  category: string;
+  id: string;
+  init_state_name: string;
+  language_code: string;
+  objective: string;
+  param_changes: IParamChangeBackendDict[];
+  param_specs: IParamSpecsBackendDict;
+  states: StatesBackendDictMapping;
+  states_schema_version: number;
+  tags: string[];
+  title: string;
+  version: number;
+  auto_tts_enabled?: boolean;
+  correctness_feedback_enabled?: boolean;
+  /* eslint-enable camelcase */
```

Comment:
We have decided to use `'` for camel_case properties. Please use that convention. Eg. 
```
'author_notes': string;
'blurb': string;
```